apps/events/models.py

```diff
@@ -18,7 +18,7 @@ class Event(models.Model):
         upload_to=event_thumbmail_path, null=True, blank=True)
 
     def __str__(self):
-        return self.title
+        return f'{self.title} - {self.description} - {self.schedule} - {self.venue}'
```

Comment:
Don't change this. We don't need this much info in `__str__`, just the title is fine. We should instead specify all the fields in the `admin.py` file which are to be displayed on the admin portal, if that's why you did this.

Reply:
ok so i will get rid of `schedule` and `venue` since u didn't ask me to put that in the main issue, but how will i get `description` tho
![image](https://user-images.githubusercontent.com/55823965/119658572-ec861680-be4a-11eb-926b-9b2ebaa831ad.png)

Reply:
@akshay399 Do you think you're getting the schedule and venue in the template from here? If yes, then you're wrong. You are passing the complete queryset to the template, and you are directly referencing the schedule, venue and description fields of Event model. So, reverting this change won't affect anything in the templates.